apps/prairielearn/src/middlewares/selectAndAuthzInstructorQuestion.ts

```diff
@@ -1,26 +1,57 @@
 import { type Request, type Response } from 'express';
 import asyncHandler from 'express-async-handler';
+import z from 'zod';
 
 import * as error from '@prairielearn/error';
 import * as sqldb from '@prairielearn/postgres';
 
+import {
+  AssessmentsFormatForQuestionSchema,
+  QuestionSchema,
+  TagsForQuestionSchema,
+  TopicSchema,
+} from '../lib/db-types.js';
+
 const sql = sqldb.loadSqlEquiv(import.meta.url);
 
+const SelectAndAuthSchema = z.object({
+  question: QuestionSchema,
+  topic: TopicSchema,
+  tags: TagsForQuestionSchema.nullable(),
+  open_issue_count: z.string(),
```

Comment:
I noticed that we have some code comparing this to a number, so I changed this to `z.coerce.number()`.

Reply:
That was fixed in 5e7766ef8772dd93f8e39314ac4eb84d77181c57.